Sources/PackageCollections/Providers/JSONPackageCollectionProvider.swift

```diff
@@ -120,6 +124,11 @@ struct JSONPackageCollectionProvider: PackageCollectionProvider {
         do {
             // This fails if collection is not signed (i.e., no "signature")
             let signedCollection = try self.decoder.decode(JSONModel.SignedCollection.self, from: data)
+
+            if !Self.enableSignatureCheck {
+                return callback(self.makeCollection(from: signedCollection.collection, source: source, signature: Model.SignatureData(from: signedCollection.signature, isVerified: true)))
```

Comment:
`isVerified: true` intentional?

Reply:
Yes, because I have slight concern of `isVerified: false` being interpreted as an error, but `isVerified: true` is more correct.

Reply:
Changed to `isVerified: false`